geth/common/types.go

```diff
@@ -257,26 +257,26 @@ type JailCell interface {
 	// Call an arbitrary JS function by name and args.
 	Call(item string, this interface{}, args ...interface{}) (otto.Value, error)
 	// Stop stops background execution of cell.
-	Stop()
+	Stop() error
 }
 
 // JailManager defines methods for managing jailed environments
 type JailManager interface {
-	// Parse creates a new jail cell context, with the given chatID as identifier.
-	// New context executes provided JavaScript code, right after the initialization.
-	Parse(chatID, js string) string
-
 	// Call executes given JavaScript function w/i a jail cell context identified by the chatID.
 	Call(chatID, this, args string) string
 
-	// NewCell initializes and returns a new jail cell.
-	NewCell(chatID string) (JailCell, error)
+	// CreateCell creates a new jail cell.
+	CreateCell(chatID string) (JailCell, error)
```

Comment:
Just curious about the reason.

Reply:
AFAIR, `NewCell` was creating or getting an already existing cell. `CreateCell` has one responsibility and using a verb is more explicit as it's not a getter.

Reply:
As I remember, no, it wasn't getting an already existing cell.
Anyway, I'm comfortable with both `NewCell` and `CreateCell`